themes/_core/js/cart.js

```diff
@@ -58,6 +58,29 @@ $(document).ready(() => {
         $input.attr('value', $input.val());
       });
 
+      if ($('.cart-payment-step-refresh').length) {
```

Comment:
Same function as `checkout-delivery.js`, should be refactor

Reply:
how about prefixing it with js like so `js-cart-payment-step-refresh`, as it is only used in js?

Reply:
@PierreRambaud indeed, how can I do that ?
@sarjon good idea, I do it